### ISSUE TITLE
propertyValueAt seems to be missing

### ISSUE THREAD

**Issue by vitormcruz** [pharo-ide/Ghost#13]
While in a debugging session of an object  proxied by ghost, the following error prompts. 

Pharo 10
Ghost version 6.0.0

`propertyValueAt:` seems to be missing

```
CompiledMethod(Object)>>doesNotUnderstand: #propertyValueAt:put:
GHInstanceMessagesInterceptor class>>prepareCannotInterpretMethodFor:
ValidationAnnouncement class(GHClassGhost)>>lookupSelector:
Context>>send:to:with:super:
Context>>send:super:numArgs:
InstructionStream>>interpretNextSistaV1InstructionFor:
EncoderForSistaV1 class>>interpretNextInstructionFor:in:
InstructionStream>>interpretNextInstructionFor:
Context>>step
[suspendedContext := suspendedContext step] in Process>>step in Block: [suspendedContext := suspendedContext step]
FullBlockClosure(BlockClosure)>>ensure:
Process>>evaluate:onBehalfOf:
Process>>step
Process>>completeStep:
DebugSession>>stepOver:
StDebuggerActionModel>>stepOver:
StDebugger>>stepOver
StStepOverCommand>>execute
SpCommand(CmCommandDecorator)>>execute
[ specCommand execute ] in [ :specCommand | 
			aButtonClass new
				label: specCommand name;
				help: specCommand description;
				in: [ :button | 
					specCommand hasIcon
						ifTrue: [ button icon: specCommand icon ] ];
				action: [ specCommand execute ];
				yourself ] in SpCommand>>configureAsButtonOfClass: in Block: [ specCommand execute ]
FullBlockClosure(BlockClosure)>>cull:
SpMorphicToolbarButtonAdapter>>execute
SpToolbarButtonMorph(PluggableButtonMorph)>>performAction:
[:m |
		(m containsPoint: evt cursorPoint) ifTrue: [m enabled ifTrue: [ m performAction: evt ]]] in SpToolbarButtonMorph(PluggableButtonMorph)>>mouseUp: in Block: [:m |...
Array(SequenceableCollection)>>do:
SpToolbarButtonMorph(PluggableButtonMorph)>>mouseUp:
SpToolbarButtonMorph(Morph)>>handleMouseUp:
MouseButtonEvent>>sentTo:
SpToolbarButtonMorph(Morph)>>handleEvent:
SpToolbarButtonMorph(Morph)>>handleFocusEvent:

```

**Comment by dionisiydk** [comment]
thanks for reports. 
One of complexity for Ghost was supporting the tooling. Now with NewTools it requires another pass.

**Comment by dionisiydk** [comment]
Fixed in v6.0.1 and Mocketry is updated as well (v7.0.1)
